### PR TITLE
Default

### PR DESCRIPTION
**Pull request by mikaelm12**
default(CancellationToken ...) -> default

### CODE REVIEW

**Comment by davidfowl** [comment]
Is this all the places?

**Comment by mikaelm12** [comment]
There are a few more. I was only looking in the HubConnectionExtensions. I can change the others here too as well

**Comment by davidfowl** [comment]
Do them all at once. All.

File: src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnectionBuilder.cs [hunk 1]
```diff
@@ -25,7 +25,7 @@ public void AddSetting<T>(string name, T value)
 
         public bool TryGetSetting<T>(string name, out T value)
         {
-            value = default(T);
+            value = default;
```
Comment (muratg) [inline src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnectionBuilder.cs]: Does this work for all `T`s that we care about?

Comment (mikaelm12) [reply src/Microsoft.AspNetCore.SignalR.Client.Core/HubConnectionBuilder.cs]: It should work just the same.

**Pull request merged**